Packs/FeedFeodoTracker/Integrations/FeedFeodoTrackerIPBlocklist/FeedFeodoTrackerIPBlocklist_description.md

```diff
@@ -10,3 +10,5 @@ There are two possible feed sources:
 - Dridex, Heodo (aka Emotet) and TrickBot botnet command & control servers (C&Cs) reside on compromised servers and servers that have been rent and setup by the botnet herder itself for the sole purpose of botnet hosting. 
 - Feodo Tracker offers a blocklist of IP addresses that are associated with such botnet C&Cs that can be used to detect and block botnet C2 traffic from infected machines towards the internet. 
 - An IP address will only get added to the blocklist if it responds with a valid botnet C2 response. However, a botnet C2 may go offline later.
+
+Relationships creation are available only for the **Last 30 Days** source.
```

Comment:
what does source mean?

Reply:
The description has an explanation regarding the two optional sources, one is a `Last 30 days` and one is the `Currently Active`. the relationships are gathered only from the first source.
Source is the term used in the beginning of the description file